documentation/CLI_usage.md

```diff
@@ -0,0 +1,195 @@
+# CLI Usage
+
+---
+## npd
+
+### Description
+napari plugin devtools scans the current python environment to find out details about napari plugin information.
+
+### Usage
+```
+npd [COMMANDS] [ARGS...]
+```
+
+### Commands
+|Command        |Description                                          |
+|---------------|-----------------------------------------------------|
+|validate       |validate the given package is correctly setup        |
+|discover       |discover all plugins under current python environment|
```

Comment:
"list" or "ls" may be shorter commands. Some things to consider are if the user would be confused about what they are listing though. I just wanted to put other options out there to see what others think.